app/javascript/packs/application.js

```diff
@@ -13,3 +13,13 @@
 
 import '../src/application.scss';
 import 'bootstrap/dist/js/bootstrap';
+import 'datatables.net';
+
+$(document).ready(function() {
```

Comment:
Can we use `document.addEventListener("DOMContentLoaded", function(){})` rather than jquery here?

Reply:
Done.